packages/manager/apps/pci-kubernetes/src/components/region-selector/KubeRegionSelector.css

```diff
@@ -0,0 +1,4 @@
+hr {
```

Comment:
You might want something more restrictive to select the element, or put this into the project css file